Dockerfile

```diff
@@ -46,10 +46,10 @@ WORKDIR /workspace/
 
 WORKDIR /tmp/
 
-# DP independent checkpoint format for distributed adam
+# Distributed Adam support for multiple dtypes
 RUN git clone https://github.com/NVIDIA/apex.git && \
   cd apex && \
-  git checkout 7995de18677295c5edeeab082179edbfdb6ee16a && \
+  git checkout 52e18c894223800cb611682dce27d88050edf1de && \
   pip3 install -v --disable-pip-version-check --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" --global-option="--fast_layer_norm" --global-option="--distributed_adam" --global-option="--deprecated_fused_adam" ./
```

Comment:
We need to make sure this Dockerfile works though.

Reply:
The Dockerfile works for me once I add the `--no-build-isolation` option when building Apex.